src/main/java/core/basesyntax/service/ReportGenerator.java

```diff
@@ -0,0 +1,9 @@
+package core.basesyntax.service;
+
+import core.basesyntax.model.FruitTransactionRow;
+import java.util.List;
+import java.util.Map;
+
+public interface ReportGenerator {
+    Map<String, Integer> calcFruitsLeftAfterTransactions(List<FruitTransactionRow> transactions);
```

Comment:
ReportGenerator should generate report. 
Go to Storage, see stock balance and generate report. That is all. Follow SRP.
```suggestion
    List<String> generateReport();
```